cime_config/testmods_dirs/allactive/wcprodssp/user_nl_eam

```diff
@@ -11,3 +11,6 @@
  fincl5 = 'PRECT','PRECC','TUQ','TVQ','QFLX','SHFLX','U90M','V90M'
  fincl6 = 'CLDTOT_ISCCP','MEANCLDALB_ISCCP','MEANTAU_ISCCP','MEANPTOP_ISCCP','MEANTB_ISCCP','CLDTOT_CAL','CLDTOT_CAL_LIQ','CLDTOT_CAL_ICE','CLDTOT_CAL_UN','CLDHGH_CAL','CLDHGH_CAL_LIQ','CLDHGH_CAL_ICE','CLDHGH_CAL_UN','CLDMED_CAL','CLDMED_CAL_LIQ','CLDMED_CAL_ICE','CLDMED_CAL_UN','CLDLOW_CAL','CLDLOW_CAL_LIQ','CLDLOW_CAL_ICE','CLDLOW_CAL_UN'
  fincl7 = 'O3', 'PS', 'TROP_P'
+
+! Specify an L80 IC to override eam.i from reference case, which is still for L72
+ ncdata = '$DIN_LOC_ROOT/atm/cam/inic/homme/eami_mam4_Linoz_ne30np4_L80_c20231010.nc'
```

Comment:
Would it be better for me to generate a L80 version of the file from the reference case?
This is very easy to do, but would still be non-BFB.

Reply:
Was it already from a reference case for v3atm? I thought you did that. If not yet, yes, it would be nice to have.  Note this is the same file you  created for #5996.

Reply:
yea, I guess it's fine to use this file. I'm just worried we will forget that this override is in place, but that's probably harmless.

Reply:
If we are moving to L80,  the default ncdata should work and not need special namelist overrides.

Reply:
It sounds like this test requires a reference case (which makes sense) which is not L72. That's why the override is used. The alternative is to remap the reference case data, but then we might still need an override to make sure the correct initial condition file is used (unless we hide the L72 version where CIME won't see it).

Reply:
Yes, the SSP tests are using a reference case (a restart point of a historical run). Since we don't have v3 historical with L80, and the reference case currently used is from v2, an override is needed for SSP tests to proceed in current code base. Much about the SSP tests will be dated once all are settled.

Reply:
Ok reference case hasn't been made yet.  Go ahead then with the override.